app/src/main/java/com/example/voyagerx/ui/fragments/settings/SettingsFragment.kt

```diff
@@ -0,0 +1,102 @@
+        createFontSizesAdapter()
+        setFontSize()
+        setBackgroundWallpaper()
+        setNotifications()
+
+        return binding.root
+    }
+
+    private fun getSharedPrefs() {
+        binding.switchAppearance.isChecked = SharedPreferencesManager.getBackgroundWallpaper()
+        binding.switchNotify.isChecked = SharedPreferencesManager.getLaunchNotifications()
+        binding.autoCompleteTextView.setText(SharedPreferencesManager.getTextInDropdown())
+
+    }
+
+    //used to populate font size dropdown with large, medium, and small selections
+    private fun createFontSizesAdapter() {
+        val fontSizes = resources.getStringArray(R.array.font_sizes)
+        val arrayAdapter = ArrayAdapter(requireContext(), R.layout.drop_down_font_sizes, fontSizes)
+        binding.autoCompleteTextView.setAdapter(arrayAdapter)
+    }
+
+    private fun setFontSize() {
+
+        //ignore - needs to be updated with styles when time permits
+/*        binding.autoCompleteTextView.setOnClickListener {
+            when (binding.autoCompleteTextView.text.toString()) {
+                "Large" -> context?.let { large -> FontSizeUtility().adjustFontScale(large, 1.0F) }
+                "Medium" -> context?.let { medium -> FontSizeUtility().adjustFontScale(medium,0.9F) }
+                "Small" -> context?.let { small -> FontSizeUtility().adjustFontScale(small, 0.8F) }
+            }
+        }*/
+
+        binding.autoCompleteTextView.setOnClickListener {
+            when (binding.autoCompleteTextView.text.toString()) {
+                "Large" -> {
+                    SharedPreferencesManager.setTextInDropdown("Large")
+                }
+                "Medium" -> {
+                    SharedPreferencesManager.setTextInDropdown("Medium")
+                }
+                "Small" -> {
+                    SharedPreferencesManager.setTextInDropdown("Small")
+                }
+            }
+        }
+    }
+
+    private fun setBackgroundWallpaper() {
+        binding.switchAppearance.setOnCheckedChangeListener { _, isChecked ->
+            if (binding.switchAppearance.isChecked) {
+                SharedPreferencesManager.setBackgroundWallpaper(isChecked)
+                Log.d("wallPaperSwitch", isChecked.toString())
+            } else {
+                SharedPreferencesManager.setBackgroundWallpaper(isChecked)
+                Log.d("wallPaperSwitch", isChecked.toString())
+            }
+        }
+    }
+
+    private fun setNotifications() {
+        binding.switchNotify.setOnCheckedChangeListener { _, isChecked ->
+            if (binding.switchNotify.isChecked) {
+                SharedPreferencesManager.setLaunchNotifications(isChecked)
+                Log.d("notificationsSwitch", isChecked.toString())
+            } else {
+                SharedPreferencesManager.setBackgroundWallpaper(isChecked)
+                Log.d("notificationsSwitch", isChecked.toString())
+            }
+        }
+    }
+
+}
```

Comment:
The notifications and the text size settings don't persist after clicking out of the settings page. I'm not sure why, but I'm happy to pair on it and figure out what's going on if needed!